packages/cached/example/persistent_storage_example/bin/gen.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:convert';
+
+import 'package:cached_annotation/cached_annotation.dart';
+import 'package:http/http.dart' as http;
+
+part 'gen.cached.dart';
+
+/// Sample API URL
+///
+/// Returns a list:
+/// [
+///   {
+///     "userId": 1,
+///     "id": 1,
+///     "title": "delectus aut autem",
+///     "completed": false
+///   },
+///   {
+///     "userId": 1,
+///     "id": 2,
+///     "title": "quis ut nam facilis et officia qui",
+///     "completed": false
+///   },
+///   198 more...
+///  ]
+const _url = 'https://jsonplaceholder.typicode.com/todos';
+
+@withCache
+abstract class Gen implements _$Gen {
+  factory Gen() = _Gen;
+
+  /// For the sake of readability, we will pass only a [persistentStorage]
+  /// here, but you can use any params combination
+  @Cached(persistentStorage: true)
+  Future<List<dynamic>> getTodos() async {
```

Comment:
```suggestion
  Future<List<Todo>> getTodos() async {
```

Please make example with classes usage not only raw json

Reply:
@Axot017 Done :)